src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -793,14 +807,15 @@ private void ReadFromBuffer(byte[] buffer, int offset, int count)
             // Do an unbuffered read directly against the underlying stream.
             Debug.Assert(_readAheadTask == null, "Read ahead task should have been consumed as part of the headers.");
             count = await _stream.ReadAsync(buffer, offset, count, cancellationToken).ConfigureAwait(false);
+            Trace("Received {0} bytes.", count);
             return count;
         }
 
         private async Task CopyFromBufferAsync(Stream destination, int count, CancellationToken cancellationToken)
         {
             Debug.Assert(count <= _readLength - _readOffset);
 
-            await destination.WriteAsync(_readBuffer, _readOffset, count, cancellationToken).ConfigureAwait(false);
+            await WriteToStreamAsync(destination, _readBuffer, _readOffset, count, cancellationToken).ConfigureAwait(false);
```

Comment:
This is a nit, but this call to WriteToStreamAsync is the only place it's not using _stream.  It seems odd to trace these the same; the others are all writing to the underlying socket, whereas this one is actually writing to the user stream.  I would trace this one with a different message, and then you can remove the Stream arg to WriteToStreamAsync as well.